sdk/core/System.Net.ClientModel/tests/perf.internal/System.Net.ClientModel.Tests.Perf.Internal.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Tests of performance System.Net.ClientModel internal types.</Description>
+    <OutputType>Exe</OutputType>
```

Comment:
You should add the perf flags that are auto added if your name ends in `.Perf`

https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.props#L18

Reply:
I'll just change this to "Tests.Internal.Perf" to get that glob-matching.